Site_Code/script.js

```diff
@@ -0,0 +1,466 @@
+      pageContribList.style.display = "";
+    }
+  }
+}
+function fixAllTabsCardsVerticalHeight(){
+  switch (previousActiveTab) {
+    case 1:
+      alignContribTableSize();
+      break;
+    case 2:
+      alignContribTableSize();
+      break;
+    case 3:
+      alignContribTableSize();
+      break;
+  }
+}
+//FINISH THIS!!
+createBannerImageWideCard("Grid-Tab-1", "home_tab_welcome_banner_image");
+createMainIntroTextCard();
+createTitlelessInfoCard();
+createHomeWideLinkCard("Grid-Tab-1", "Survival Guide", "A website where we host tips, tricks and guidance for those who are attempting the Cyber Discovery programme. All these articles have been written by past participants, most of whom have progressed to the Elite stage themselves.", "cdsurvivalguide_home_card", "https://cdsurvivalguide.netlify.com/", "View");
+createHomeWideLinkCard("Grid-Tab-1", "Discord Server", "Join the largest Cyber Discovery Discord server with nearly 750 like-minded people and students who are going through the programme as well as a few SANS Institute staff members who are involved in running the programme!", "home_discord_invite_card", "http://discord.gg/Kf8n5rT", "Join");
+createHomeWideLinkCard("Grid-Tab-1", "CyberStart Assess", "The first stage of year 2 is now open! Register now, or alternatively, if you have an existing account you can simply just sign back in here using your details from last year. CyberStart Assess will remain open till the 7th of January 2019.", "home_assess_card_reminder", "https://assess.joincyberdiscovery.com/reg-login", "Go");
+createHomeWideLinkCard("Grid-Tab-1", "Medium Blog", "View the official Cyber Discovery medium blog. This is where they post different announcements, challenges, student spotlights, answers, writeups, and much more. You can also download our app which has the medium blog embedded within it!", "medium_blog_main_card_image", "https://medium.com/@CyberDiscUK", "Read");
+createLinkCard("Grid-Tab-5", "OverTheWire", "Wargames offered by the OverTheWire community, which can help you to learn and practice security concepts in the form of fun-filled games.", "otw-card-background", "https://overthewire.org/wargames/");
+createLinkCard("Grid-Tab-5", "Replit", "A free, powerful and simple online compiler, IDE, interpreter, and REPL. Allows you to Code, compile, and run code in 30+ programming languages.", "replit-card-background", "https://repl.it/languages");
+createLinkCard("Grid-Tab-5", "SmashTheStack", "An ethical hacking environment that supports the simulation of real world software vulnerabilities and allows the use of exploitation techniques.", "smashthestack-card-background", "http://smashthestack.org/wargames.html");
+createLinkCard("Grid-Tab-5", "Cybrary", "Online cyber-security training which you can learn anytime, anywhere with open-source, high quality training from top professionals and companies.", "cybrary-card-background", "https://www.cybrary.it/");
```

Comment:
```suggestion
createLinkCard("Grid-Tab-5", "Cybrary", "Online cyber security training which you can learn anytime, anywhere with open-source, high quality training from top professionals and companies.", "cybrary-card-background", "https://www.cybrary.it/");
```
We need to standardise this.